### ISSUE TITLE
Issue with reflection and empty schema in ORM model

### ISSUE THREAD

**Issue by EtienneDG** [geoalchemy/geoalchemy2#442]
Hello there,

I tripped upon an issue using MySQL with the latest commit from master.

My context is the following :
I have 2 Mysql databases : `db_dev` and `db_test` (you guessed their usage from their names).
I never specify a shema name in my models, which is fine with SQLAlchemy that uses other ways to get the schema name if needed.
I run `alembic check` on CI to ensure that I don't have diff between my model and an up-to-date dummy DB.

When reflecting a column for MySQL (the code looks similar for postgre, but I don't know if it causes the same issue) GeoAlchemy2  adds an `AND` clause on `INFORMATION_SCHEMA.COLUMNS:SCHEMA_NAME` IF the schema of the table is not None. 

https://github.com/geoalchemy/geoalchemy2/blob/aac6d078d23dc7c6153a9fc13383e925f0dfe7e1/geoalchemy2/admin/dialects/mysql.py#L34-L40

In my case, this causes the `geometry_type_query` to return two lines : 
- one for the column in the `db_dev`
- one for the column in the `db_test`

And further down the line, causes SQLAlchemy to raise this error : 
```
  File "/home/.../lib/python3.10/site-packages/geoalchemy2/admin/__init__.py", line 99, in _reflect_geometry_column
    select_dialect(inspector.bind.dialect.name).reflect_geometry_column(
  File "/home/.../lib/python3.10/site-packages/geoalchemy2/admin/dialects/mysql.py", line 41, in reflect_geometry_column
    geometry_type, srid, nullable_str = inspector.bind.execute(text(geometry_type_query)).one()
  File "/home/.../lib/python3.10/site-packages/sqlalchemy/engine/result.py", line 1258, in one
    return self._only_one_row(
  File "/home/.../lib/python3.10/site-packages/sqlalchemy/engine/result.py", line 614, in _only_one_row
    raise exc.MultipleResultsFound(
sqlalchemy.exc.MultipleResultsFound: Multiple rows were found when exactly one was required
```

A fix could be to default `schema` to `inspector.default_schema_name` if the schema of the current table is `None`, inside the `reflect_geometry_column` method.

I can propose a PR if it helps.

**Comment by adrien-berchet** [comment]
Very interesting, thank you very much for this report!
Indeed, I guess we should always check the default schema, even for other dialects. If you have some time to propose a PR for this it would be very helpful, so you can test it on your own test case directly.

**Comment by adrien-berchet** [comment]
Fixed by #443
